src/ontology/cl-edit.owl

```diff
@@ -30532,7 +30532,7 @@ SubClassOf(obo:CL_4023164 obo:CL_4023162)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22865660") obo:IAO_0000115 obo:CL_4023168 "A neuron that is part of the somatic sensory system. Somatosensory neurons innervate the skin or integument to detect different types of thermal, chemical, and mechanical touch stimuli.")
 AnnotationAssertion(terms:contributor obo:CL_4023168 <http://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(rdfs:label obo:CL_4023168 "somatosensory neuron")
```

Comment:
One of the child terms "Rohon-Beard neuron" seems to not fit the new classification as a peripheral sensory neuron

Reply:
You are correct, this neuron should actually be under primary neuron (sensu Teleostei) this will also require another ticket to classify the Teleostei neurons.

Reply:
The definition of somatosensory neurons will require further modifications, as somatosensory neurons per se are those cell types that receive the external stimulus and covert it into electrical input. I can leave somatosensory neurons out of the peripheral nervous system classification and open another ticket to understand where to categorise them.